src/internal/git/push.go

```diff
@@ -32,7 +32,7 @@ func PushAllDirectories(localPath string) error {
 	}
 
 	spinner := message.NewProgressSpinner("Processing %d git repos", len(paths))
-	defer spinner.Stop()
+	defer spinner.Success()
```

Comment:
If this function returns at line 42 or 52 this would erroneously say it succeeded.

Reply:
same, will revert this for now.